### PR TITLE
Add workaround to display TIMESTAMP_LTZ properly

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

**Review by cbrswt — APPROVED** [review]

File: streaming_jupyter_integrations/cast_utils.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+def __contains_timestamp_ltz_fields(table: Table) -> bool:
+    table_schema = table.get_schema()
+    return any(__is_timestamp_ltz(t) for t in table_schema.get_field_data_types())
+
+
+def __is_timestamp_ltz(field_type: DataType) -> bool:
+    return isinstance(field_type, LocalZonedTimestampType)
+
+
+def __cast_timestamp_ltz_to_string(tenv: TableEnvironment, table: Table) -> 'Table':
```
Comment (andrzejdackiewicz) [inline streaming_jupyter_integrations/cast_utils.py:24]: Just a proposal :smile_cat: Perhaps `__add_string_cast_for_timestamp_ltz_fields_in_table` would be a more adequate name for the logic of this function?

Comment (grzegorz8) [reply streaming_jupyter_integrations/cast_utils.py:24]: Hmm, IMO the name proposed is a bit too long :P

Comment (andrzejdackiewicz) [reply streaming_jupyter_integrations/cast_utils.py:24]: :) Besides that LGTM!

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM!

**Pull request merged**